README.md

```diff
@@ -6,5 +6,9 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+It exists so that a server can process a request from the client, speak to a controller, which speaks to a model, which interacts with a database IN ORDER TO translate the request to the proper data, to return that to the controller, to return that to the server, which sends that to the client in the proper format with the proper data. \sigh
```

Comment:
That's more of a 'what' than a 'why'. Rails exists to make it easier to quickly spin up web applications by providing us with an architecture and pre-built tools.